README.md

```diff
@@ -409,11 +409,9 @@ go mod download
 Update the generated files,
 
 ```bash
-go generate
+go generate ./...
```

Comment:
Can you please explain the significance of `./...`? Because I ran `go generate` from the root of the repo on the `oapi-codegen-upgrade` branch with the v2 changes and it works fine.

Reply:
The `go generate` command accepts a list of packages on which it should operate, or if none are listed then it operates on the package in the current directory, without descending into any subpackages. Running in the root of the project therefore will operate on the `typesense` package but not the `api`, `mocks` or `test` subpackages. The ellipsis can be used in package specifiers as a wildcard to match any subpackage paths, so using it will run generation tools declared in any of the subpackages in the project.

https://pkg.go.dev/cmd/go#hdr-Package_lists_and_patterns

You should see a difference in the amount of output when running `go generate` with and without the `./...` package list argument.

Reply:
Also, note that you'll really need to use the resulting codebase after merging both the mockgen change and the oapi-codegen change to see the full behaviour of `go generate ./...`, otherwise it's likely that only one of the two codegeneration tools will actually work, unless you have the tool binary in your shell's path.

Reply:
Got it. I just realized that I had merged the mockgen PR into master branch but the other PR into the temporary branch I had created. Now, as I merge that temp branch changes into master, I see this conflict: https://gist.github.com/kishorenc/4577ff0acb18ae8f0ddeabf4987eb3ac

Reply:
👍 to resolve those conflicts I would choose the highest version of each of the conflicting indirect dependencies, that will satisfy the requirements of both tool packages.